dns/push.sh

```diff
@@ -65,4 +83,21 @@ if [ $? != 0 ]; then
 fi
 echo "Prod push SUCCEEDED"
 
-# TODO: run test against prod
+echo "Testing prod zones"
+./check-zone.sh "k8s.io." > log.prod 2>&1
+if [ $? != 0 ]; then
+    echo "Canary test FAILED, halting; log follows:"
+    echo "========================================="
+    cat log.canary
+    exit 2
+fi
+./check-zone.sh "kubernetes.io." > log.prod 2>&1
+if [ $? != 0 ]; then
+    echo "Canary test FAILED, halting; log follows:"
+    echo "========================================="
+    cat log.canary
+    exit 2
+fi
+echo "Canary test SUCCEEDED"
```

Comment:
s/Canary/Prod

Reply:
```
Dry-run to canary zones
Pushing to canary zones
Canary push SUCCEEDED
Testing canary zones
Canary test SUCCEEDED
Dry-run to prod zones
Pushing to prod zones
Prod push SUCCEEDED
This will likely fail the first time through... takes a bit to propogate
Testing prod zones
Canary test SUCCEEDED
Prod test SUCCEEDED
```